backend/dataall/modules/datasets/cdk/dataset_custom_resources_extension.py

```diff
@@ -0,0 +1,239 @@
+import os
+import logging
+import pathlib
+from aws_cdk import (
+    custom_resources as cr,
+    aws_iam as iam,
+    aws_lambda as _lambda,
+    aws_lambda_destinations as lambda_destination,
+    aws_ssm as ssm,
+    aws_sqs as sqs,
+    aws_kms as kms,
+    RemovalPolicy,
+    Duration,
+    CustomResource,
+)
+
+from dataall.core.environment.cdk.environment_stack import EnvironmentSetup, EnvironmentStackExtension
+
+log = logging.getLogger(__name__)
+
+
+class DatasetCustomResourcesExtension(EnvironmentStackExtension):
+    """Extends an environment stack for LakeFormation settings custom resource and Glue database custom resource"""
+
+    @staticmethod
+    def extent(setup: EnvironmentSetup):
+        kms_key = DatasetCustomResourcesExtension.set_cr_kms_key(
+            setup=setup,
+            environment=setup.environment(),
+            group_roles=setup.group_roles,
+            default_role=setup.default_role
+        )
+
+        # Lakeformation default settings custom resource
+        # Set PivotRole as Lake Formation data lake admin
+        entry_point = str(
+            pathlib.PosixPath(os.path.dirname(__file__),
+                              './assets/lakeformationdefaultsettings').resolve()
```

Comment:
In `DatasetGlueProfilerExtension`, the relative path is different. We need to check which one is correct

Reply:
I tested the creation of the Dataset with the extensions and it worked fine with this paths. At first I had the same level for both but i think because of the libraries we use they are defined diferently